api/grpcserver/mesh_service.go

```diff
@@ -357,15 +361,27 @@ func (s MeshService) readLayer(layer *types.Layer, layerStatus pb.Layer_LayerSta
 }
 
 // LayersQuery returns all mesh data, layer by layer
-func (s MeshService) LayersQuery(ctx context.Context, in *pb.LayersQueryRequest) (*pb.LayersQueryResponse, error) {
+func (s MeshService) LayersQuery(_ context.Context, in *pb.LayersQueryRequest) (*pb.LayersQueryResponse, error) {
 	log.Info("GRPC MeshService.LayersQuery")
 
+	var startLayer, endLayer types.LayerID
+	if in.StartLayer == nil {
+		startLayer = 0
+	} else {
+		startLayer = types.LayerID(in.StartLayer.Number)
+	}
+	if in.EndLayer == nil {
+		endLayer = 0
```

Comment:
what does this mean? no values will be returned?

Reply:
yup